APNS/APNS/APNSPusher.swift

```diff
@@ -24,11 +24,11 @@ public final class APNSPusher: NSObject, APNSPushable {
             switch type {
             case .certificate(let _identity):
                 identity = _identity
-                session = URLSession(configuration: .default,
+                session = URLSession(configuration: .ephemeral,
```

Comment:
Is it for privacy purpose?

Reply:
I was thinking more about disabled caching

Reply:
Ok fine for me 👍